examples/ddi_seal/config.py

```diff
@@ -0,0 +1,59 @@
+from yacs.config import CfgNode as CN
+import os
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Basic
+# -----------------------------------------------------------------------------
+_C.OUTPUT_DIR = 'results'
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.ROOT = "dataset/ogbl_ddi"
+_C.DATASET.NAME = 'ogbl-ddi'
+_C.DATASET.NUM_HOPS = 1
+_C.DATASET.TRAIN_PERCENT = 1.0
```

Comment:
Is 1% a good/common default?

Reply:
This is the default setting in original execution (leaderboard). As ddi is a high-degree graph, SEAL actually transfers link prediction task to a graph classification, therefore the default 1.0 is enough to for high degree graph training in this ddi example.

Reply:
I'll also add some instructions on it. This is not training dataset sample rate, instead it's the link' graph construction neighbors' sample rate.